test/parallel/test-net-server-close-before-calling-lookup-callback.js

```diff
@@ -3,5 +3,5 @@
 const common = require('../common');
 const net = require('net');
 // Process should exit because it does not create a real TCP server.
-// Paas localhost to ensure create TCP handle asynchronously because It causes DNS resolution.
+// Pass localhost to ensure create TCP handle asynchronously because It causes DNS resolution.
```

Comment:
```suggestion
// Pass localhost to ensure create TCP handle asynchronously because it causes DNS resolution.
```

If your updating this file, might as well fix the other typo anyway.